tests/kafkatest/services/kafka/config.py

```diff
@@ -25,7 +25,8 @@ class KafkaConfig(dict):
         config_property.PORT: 9092,
         config_property.SOCKET_RECEIVE_BUFFER_BYTES: 65536,
         config_property.LOG_DIRS: "/mnt/kafka/kafka-data-logs-1,/mnt/kafka/kafka-data-logs-2",
-        config_property.ZOOKEEPER_CONNECTION_TIMEOUT_MS: 2000
+        config_property.ZOOKEEPER_CONNECTION_TIMEOUT_MS: 5000,
+        config_property.ZOOKEEPER_SESSION_TIMEOUT_MS: 6000
```

Comment:
The default ZK session timeout and ZK connection timeout are both 18 secs. So we can just change both to 18secs here.

Reply:
sure!